### ISSUE TITLE
What is the MAE after trainning?

### ISSUE THREAD

**Issue by xjtulyc** [fqhank/Ordinal-Regression-for-Age-Estimation#1]
Hello!

I am trying to implement this work. However, the MAE is 4.5276 after trainning the model. How to imporve the MAE metric like the paper?

Thanks.

**Comment by fqhank** [comment]
hello!
thanks for asking.firstly, I believe that the MAE you've achieved is way too large, I strongly suggest you to check your implementation of input data, the result after training without designed strategy should be around 3.6-3.7.then, implement stronger augmentation and change fine-tune your optimizer would further improve your result.looking forward to your good news.
best wishes

**Comment by xjtulyc** [comment]
> hello! thanks for asking.firstly, I believe that the MAE you've achieved is way too large, I strongly suggest you to check your implementation of input data, the result after training without designed strategy should be around 3.6-3.7.then, implement stronger augmentation and change fine-tune your optimizer would further improve your result.looking forward to your good news. best wishes

Thanks! I apply kaiming initialization to the neural network, and get better performance. About the input data problem, I have delete some image that cannot be read, any problem I should pay attention to it?
